torch/_inductor/codegen/common.py

```diff
@@ -1738,7 +1738,9 @@ def inner(*args, **kwargs):
                     value = getattr(parent_handler, name)(*args, **kwargs)  # type: ignore[has-type]
```

Comment:
Thanks for the fix. For ops name like `lt`, `getattr(parent_handler, name)` will back trace until `MockHandler` to do the codegen. So, for this case of nested kernel, it back trace to use instance of `CSEProxy` binding with parent kernel (`CppTemplateKernel` in this case).